package.json

```diff
@@ -18,13 +18,15 @@
     "test:codecov-upload": "nyc report --reporter=json && codecov -f coverage-report/*.json",
     "sonar-scanner": "sonar-scanner -Dsonar.projectName='CMC :: Citizen Frontend' -Dsonar.sources=src/main -Dsonar.tests=src/test -Dsonar.exclusions=src/main/public/** -Dsonar.typescript.lcov.reportPaths=coverage-report/lcov.info",
     "precommit": "yarn lint",
-    "prepush": "yarn test && yarn test:routes"
+    "prepush": "yarn test && yarn test:routes",
+    "snyk-protect": "snyk protect --debug",
+    "prepare": "yarn snyk-protect"
```

Comment:
I'm worried that this will slow down the pipeline as we run `yarn install` a number of times during it.
It's something that we would like to run locally but not on Jenkins I would think?